Project.toml

```
@@ -1,11 +1,30 @@
 name = "UnfoldRIDE"
 uuid = "348b5554-d1f6-52bd-94b1-729601d351bf"
-authors = [
-    "Till Prölß",
-    " René Skukies",
-    " Benedikt Ehinger",
-]
+authors = ["Till Prölß", " René Skukies", " Benedikt Ehinger"]
 version = "0.1.0"
 
+[deps]
+CairoMakie = "13f3f980-e62b-5c42-98c6-ff1f3baf88f0"
+DSP = "717857b8-e6f2-59f4-9121-6e50c889abd2"
+DataFrames = "a93c6f00-e57d-5684-b7b6-d8193f3e46c0"
+DataFramesMeta = "1313f7d8-7da2-5740-9ea0-a2ca25f37964"
+Distributions = "31c24e10-a181-5473-b8eb-7969acd0382f"
+FFTW = "7a1cc6ca-52ef-59f5-83cd-3a7055c09341"
+Parameters = "d96e819e-fc66-5662-9728-84c9c7592b0a"
+Peaks = "18e31ff7-3703-566c-8e60-38913d67486b"
+Random = "9a3f8284-a2c9-5f02-9a11-845980a1fd5c"
+Revise = "295af30f-e4ad-537b-8983-00126c2a3abe"
+StableRNGs = "860ef19b-820b-49d6-a774-d7a799459cd3"
+Statistics = "10745b16-79ce-11e8-11f9-7d13ad32a3b2"
+Unfold = "181c99d8-e21b-4ff3-b70b-c233eddec679"
+
 [compat]
+CairoMakie = "0.13.1"
+Distributions = "0.25.117"
+FFTW = "1.8.1"
+Parameters = "0.12.3"
+Peaks = "0.5.3"
```

```suggestion
Peaks = "0.5"
```
don't fix on bugfix versions, but feature / major versions